src/store/actions.ts

```diff
@@ -251,20 +251,21 @@ const commitExistingData = ({ commit, getters }): void => {
   }
   commit('mutateEntityType', entity_type_cd)
   if (!getters.getEntityTypeOptions.some(option => option.value === entity_type_cd)) {
-    let obj = getters.getEntityTypesBC.find(entity => entity.value === entity_type_cd)
+    const obj = getters.getEntityTypesBC.find(entity => entity.value === entity_type_cd)
       ? getters.getEntityTypesBC.find(entity => entity.value === entity_type_cd)
       : getters.getEntityTypesXPRO.find(entity => entity.value === entity_type_cd)
     commit('mutateEntityTypeAddToSelect', obj)
   }
-  let { requestTypeCd, request_action_cd } = getters.getNr
+  const { requestTypeCd } = getters.getNr
+  let { request_action_cd } = getters.getNr
```

Comment:
ESLint complained about `requestTypeCd` needing to be a const, but `request_action_cd` cannot be const, so I broke up this declaration like this.